src/shell.js

```diff
@@ -201,7 +201,11 @@ if (ENVIRONMENT_IS_NODE) {
   process['on']('unhandledRejection', abort);
 #endif
 
-  quit_ = function(status) {
+  quit_ = function(status, toThrow, keepalive) {
+    if (keepalive) {
```

Comment:
That is, I am proposing that this be
```suggestion
    if (keepRuntimeAlive()) {
```

Reply:
Done